layout/_macro/post-end-tag.swig

```diff
@@ -0,0 +1,5 @@
+<div>
+  {% if theme.post_end_tag.enabled and theme.post_end_tag.message %}
+    <div style="text-align:center;color: #ccc;font-size:14px;">{{ theme.post_end_tag.message }}</div>
```

Comment:
And remove the hard-coded `font-size` here. Let it inherit from the parent element. See: https://github.com/theme-next/hexo-theme-next/pull/800